src/utils/MySQLDataHandler.py

```diff
@@ -14,6 +14,23 @@
 
 from prometheus_client import Histogram
 
+# sys.path.append("../ClusterBootstrap")
```

Comment:
duplicated codes? consider to move it to the utils.py or somewhere similar so that we can keep it consistent in one place.